packages/vsphere/changelog.yml

```
@@ -1,10 +1,13 @@
 # newer versions go on top
 - version: "1.15.0-next"
```

```suggestion
- version: "1.15.0"
```
As this PR is the last one we need to remove `next` tag and update version in manifest to `1.15.0`.

---

I guess so, should we do it with this PR @ishleenk17?

---

Lets keep the next label as of now.
We will raise a PR when we are ready to release it.